lib/src/pubspec_parse.dart

```diff
@@ -37,9 +37,9 @@ abstract class PubspecBase {
         _version = version;
 
   /// The package's name.
-  String get name {
-    if (_name != null) return _name!;
+  String get name => _name ??= _lookupName();
```

Comment:
`late final`.

Reply:
also not possible here, it can be set in the constructor